modules/tinymce/changelog.txt

```diff
@@ -1,17 +1,17 @@
 Version 5.4.2 (TBD)
-    Fixed resizing the window not resizing the editor in fullscreen mode #TINY-3511
+    Fixed the editor not resizing when resizing the browser window in fullscreen mode #TINY-3511
     Fixed clicking on notifications causing inline editors to hide #TINY-6058
-    Fixed an issue where link urls couldn't be deleted or edited in the link dialog in some cases #TINY-4706
-    Fixed a regression where the `anchor_top` and `anchor_bottom` settings weren't working when set to `false` #TINY-6256
-    Fixed the `anchor` plugin not supporting the `allow_html_in_named_anchor` setting #TINY-6236
+    Fixed an issue where link URLs could not be deleted or edited in the link dialog in some cases #TINY-4706
+    Fixed a regression where setting the `anchor_top` or `anchor_bottom` options to `false` was not working #TINY-6256
+    Fixed the `anchor` plugin not supporting the `allow_html_in_named_anchor` option #TINY-6236
     Fixed an exception thrown when removing inline formats that contained additional styles or classes #TINY-6288
     Fixed an exception thrown when positioning the context toolbar on Internet Explorer 11 in some edge cases #TINY-6271
     Fixed inline formats not removed when more than one `removeformat` format rule existed #TINY-6216
     Fixed an issue where spaces were sometimes removed when removing formating on nearby text #TINY-6251
-    Fixed list toolbar buttons not showing as active when a list is selected #TINY-6286
-    Fixed an issue where the UI would sometimes not be shown or hidden when calling show or hide methods on the editor #TINY-6048
+    Fixed the list toolbar buttons not showing as active when a list is selected #TINY-6286
+    Fixed an issue where the UI would sometimes not be shown or hidden when calling the show or hide API methods on the editor #TINY-6048
     Fixed the list type style not retained when copying list items #TINY-6289
-    Fixed tabs in plain text converted to a single space and added new `paste_tab_spaces` setting to control how many spaces are used to represent a tab #TINY-6237
+    Fixed the Paste plugin converting tabs in plain text to a single space character with a new `paste_tab_spaces` option for setting the number spaces used to replace a tab character #TINY-6237
```

Comment:
number _of_ spaces

Also, that's a really big sentence to read without any real pauses

Reply:
updated by https://github.com/tinymce/tinymce/pull/5968/commits/bf7e8f9b87b27f16dca6fb80f727342b1b4d9fd0